config/var_development.env

```diff
@@ -2,6 +2,8 @@ DOCS_BASE_PATH=/api
 DOCS_EXTERNAL_DESCRIPTION="Documentation for using the Platform APIs"
 DOCS_EXTERNAL_URL=https://docs.google.com/document/d/1p3q9OT9latXqON20WDh4CNPxIShUunfGgqT163r-Caw/edit?usp=sharing
 DOCS_HOST=dev-platform.nypl.org
+DOCS_INFO_DESCRIPTION="Endpoints require an OpenID Connect token from the NYPL OAuth server and interact with NYPL Data Streams."
+DOCS_INFO_TITLE="NYPL Platform APIs"
```

Comment:
We had thought that DOCS_INFO_DESCRIPTION and DOCS_INFO_TITLE should only be in var_app, since they are common to everything. Is that not the case?